dkeyserver/Enclave/enclave.cpp

```diff
@@ -645,13 +645,13 @@ sgx_status_t sgx_get_domainkey(uint8_t *domain_key,
 
```

Comment:
commit msg:
"
Fix a issue of strncmp in dkeyserver

test: passed all the unit test
Signed-off-by: hongjingmao <hongjing.mao@intel.com>
"

Reply:
done.